client.go

```diff
@@ -318,9 +378,14 @@ func (c *Client) Start() (addr net.Addr, err error) {
 	{
 		cmdSet := c.config.Cmd != nil
 		attachSet := c.config.Reattach != nil
+		secureSet := c.config.SecureConfig != nil
 		if cmdSet == attachSet {
 			return nil, fmt.Errorf("Only one of Cmd or Reattach must be set")
 		}
+
+		if secureSet && attachSet {
+			return nil, fmt.Errorf("only one of Reattach or SecureConfig can be set")
+		}
```

Comment:
I may have missed it but is there a test to cover this? Should be pretty simple but would be worth covering since the implication is high.